app/models/rent.rb

```diff
@@ -252,24 +230,24 @@ def p_email
   def as_json(options = {})
     if service
       {
-          :id => self.id,
-          :title => "Service",
-          :start => self.d_from.rfc822,
-          :end => self.d_til.rfc822,
-          :status => self.comment,
-          :backgroundColor => "black",
-          :textColor => "white"
+          id: id,
+          title: 'Service',
+          start: d_from.iso8601,
+          end: d_til.iso8601,
+          status: comment,
+          backgroundColor: 'black',
+          textColor: "white"
       }
     else
       {
-          :id => self.id,
-          :title => self.p_name,
-          :start => self.d_from.rfc822,
-          :end => self.d_til.rfc822,
-          :url => self.p_path,
-          :status => self.status,
-          :backgroundColor => backgroundColor(self.status, self.aktiv),
-          :textColor => "black"
+          id: id,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.